db/migrate/20170530102506_add_deleted_to_containers_tables.rb

```diff
@@ -0,0 +1,25 @@
+class AddDeletedToContainersTables < ActiveRecord::Migration[5.0]
+  def change
+    add_column :container_definitions, :deleted, :boolean, :default => false, :null => false
+    add_column :container_groups, :deleted, :boolean, :default => false, :null => false
+    add_column :container_images, :deleted, :boolean, :default => false, :null => false
+    add_column :container_projects, :deleted, :boolean, :default => false, :null => false
+    add_column :containers, :deleted, :boolean, :default => false, :null => false
```

Comment:
Is a default_value_for a ruby thing, done on Model? It make sense to have this ensured on the DB layer.